run_route_scripts/run_with_server.py

```diff
@@ -85,7 +85,8 @@ def make_request(post_body):
   parser.add_argument('--output-dir', type=str, help='The directory in which to place the result of each request')
   parser.add_argument('--concurrency', type=int, help='The number of processes to use to make requests', default=multiprocessing.cpu_count())
   parser.add_argument('--format', type=str, help='Supports csv, json, raw and null output formats', default='csv')
-  parser.add_argument('--headers', type=str, help='Additional http headers to send with the requests. Follows the http header spec, eg. some-header-name: some-header-value', action='append', nargs='*')
+  parser.add_argument('--headers', type=str, help='Additional http headers to send with the requests. Follows the http\
```

Comment:
This accidentally crept in. This script is currently broken in master. But I remember a PR from either @kevinkreiser or @merkispavel that fixed it (can't find the PR right now though :/ )

Reply:
its over in @genadz 's pr

Reply:
ah ok :facepalm: